app/(searching-consultant)/search.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { StyleSheet, ScrollView, View } from 'react-native';
+
+import { theme } from '../../styles/theme';
+import { useSpecialities } from '../../context/FilterConsultantsContext';
+import SearchBar from '../../components/common/SearchBar';
+import FilterTabs from '../../components/searching-consultant/tabs/FilterTabs';
+import ConsultantCard from '../../components/searching-consultant/consultant-card/ConsultantCard';
+import useConsultants from '../../hooks/useConsultants';
+import placeHolder from '../../locales/en/SearchBar.json';
+
+const { consultingSearching } = placeHolder;
+
+export default function SearchConsultant() {
+	const [searchQuery, setSearchQuery] = React.useState(''); // holds the state for the search Query
+
+	const { selectedSpecialities, experience, selectedRegions, price } =
+		useSpecialities(); // Context that holds all current filtering states
+
+	const { filteredConsultants, isLoading, fetchError } = useConsultants(
+		searchQuery,
+		selectedSpecialities,
+		experience,
+		selectedRegions,
+		price,
+	); //Loads in the data and filters the data through a customHook
+
+	const onSearch = (query) => {
+		setSearchQuery(query);
+	}; //search Query that allows us to search for consultants
+
+	return (
+		<ScrollView style={styles.containerFull}>
+			<View style={styles.container}>
+				<SearchBar
+					searchQuery={searchQuery}
+					onSearch={onSearch}
+					placeHolder={consultingSearching}
+				/>
+			</View>
+			<FilterTabs />
+			<ConsultantCard
```

Comment:
Nitpick: `ConsultantCard` should be renamed to something that indicates the plurality of consultants (maybe `ConsultantList` or `ConsultantCards`). By making this change, other devs will immediately know which part of the UI corresponds with the React component

Reply:
Nitpicking is totally fine @quanphan2906 . I want for my code to be clear to other developers. Keep at it. I will make this change